raw-data/EA/EA_inventories.tsv

```diff
@@ -8112,36 +8112,36 @@ InventoryID	LanguageCode	LanguageName	Segment
 2433	kjh	Khakas (Standard)	uː
 2433	kjh	Khakas (Standard)	ɔ
 2433	kjh	Khakas (Standard)	ɔː
-2434	0	Eastern Khanty (Vakh)	p
-2434	0	Eastern Khanty (Vakh)	v
-2434	0	Eastern Khanty (Vakh)	m
-2434	0	Eastern Khanty (Vakh)	t
-2434	0	Eastern Khanty (Vakh)	s
-2434	0	Eastern Khanty (Vakh)	n
-2434	0	Eastern Khanty (Vakh)	l
-2434	0	Eastern Khanty (Vakh)	ʈʂ
-2434	0	Eastern Khanty (Vakh)	ɳ
-2434	0	Eastern Khanty (Vakh)	ɭ
-2434	0	Eastern Khanty (Vakh)	r
-2434	0	Eastern Khanty (Vakh)	c
-2434	0	Eastern Khanty (Vakh)	j
-2434	0	Eastern Khanty (Vakh)	ɲ
-2434	0	Eastern Khanty (Vakh)	ʎ
-2434	0	Eastern Khanty (Vakh)	q
-2434	0	Eastern Khanty (Vakh)	ʁ
-2434	0	Eastern Khanty (Vakh)	ŋ
-2434	0	Eastern Khanty (Vakh)	iˑ
-2434	0	Eastern Khanty (Vakh)	eˑ
-2434	0	Eastern Khanty (Vakh)	æˑ
-2434	0	Eastern Khanty (Vakh)	yˑ
-2434	0	Eastern Khanty (Vakh)	øˑ
-2434	0	Eastern Khanty (Vakh)	ɯˑ
-2434	0	Eastern Khanty (Vakh)	ɑˑ
-2434	0	Eastern Khanty (Vakh)	oˑ
-2434	0	Eastern Khanty (Vakh)	ɪ
-2434	0	Eastern Khanty (Vakh)	ʏ
-2434	0	Eastern Khanty (Vakh)	ɯ̽
-2434	0	Eastern Khanty (Vakh)	ʊ
+2434	NA	Eastern Khanty (Vakh)	p
```

Comment:
Is there a reason that we leave this as `NA` rather than assigning it to the ISO code for Khanty in general?  In other words, if we're using ISO codes as our primary classification mechanism, on what grounds are we making the judgment that ISO is wrong about this being a dialect of `kca`, but is right about, say, Darkhat being a dialect of `khk`?